tests/functional/test_events.py

```diff
@@ -112,8 +113,8 @@ def service(sdk_client_fs, name='zookeeper'):
     return cluster(sdk_client_fs).service_add(name=name)
 
 
-def cluster_action_run(sdk_client_fs, name):
-    return cluster(sdk_client_fs).action(name=name).run()
+def cluster_action_run(sdk_client_fs, name, verbose_state=False):
+    return cluster(sdk_client_fs).action(name=name).run(verbose=verbose_state)
```

Comment:
It might make sense to use `**kwargs` to pass parameters to run method. Now there is only verbose, but tomorrow in a new test may need to send `config` or `config_diff`

```suggestion
def cluster_action_run(sdk_client_fs, name, **kwargs):
    return cluster(sdk_client_fs).action(name=name).run(**kwargs)
```

And in tests:
`cluster_action_run(sdk_client_fs, name="install", verbose=verbose_state)`